include/elliptics/packet.h

```diff
@@ -1145,6 +1218,21 @@ static inline void dnet_convert_monitor_stat_request(struct dnet_monitor_stat_re
 	r->categories = dnet_bswap32(r->categories);
 }
 
+
+struct dnet_server_send_request {
+	int		id_num;
+	int		group_num;
+	uint64_t	iflags;
+};
```

Comment:
No, it should not, since it is already evenly packed to the highest member size.

Packing is a very bad idea generally, unaligned access is very expensive on the most but x86 architectures, it should only be used when we absolutely have to because of badly designed structures.